reactful-numbers/src/app/components/ButtonsBox.js

```diff
@@ -1,67 +1,168 @@
 import React from "react";
 import Button from "./Button";
-function ButtonsBox() {
+
+function ButtonsBox({
+  setInputValue,
+  clearInput,
+  invertSign,
+  percentage,
+  expressionBoxValue,
```

Comment:
The value of the expressionBox is needed for some calculations and checks from button pressing like to check if the expressionsBox input field is empty and depending on that the clear buttons render differently.